_episodes/07-branches.md

```diff
@@ -80,13 +101,13 @@ $ git branch
 ~~~
 {: .output}
 
+Before we used the `checkout` command to checkout a commit using
```

Comment:
That is very true. I wasn't quite sure how to approach that either. I'd like to address that somehow because the `git checkout` command is now being used to do several different things.